### PR TITLE
HPCC-14623 Allow SUM(GROUP)/y inside a TABLE activity

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@jamienoss please review.  It is much easier to compare if you ignore whitespace changes.

**Comment by AttilaVamos** [comment]
The missing test result is my fault. I will fix it and reschedule this PR.

File: ecl/hqlcpp/hqlttcpp.cpp [hunk 1]
```diff
@@ -1506,207 +1506,221 @@ static IHqlExpression * createArith(node_operator op, ITypeInfo * type, IHqlExpr
...
+                OwnedHqlExpr n11 = createValue(no_sqrt, LINK(exprType), LINK(n10));
+                OwnedHqlExpr n12 = createArith(no_div, exprType, n3, n11);
+                return normalizeAggregateExpr(selector, n12, false);
+            }
+            throwUnexpected();
+
+        case no_variance:
+        case no_covariance:
+        case no_correlation:
+        case no_field:
```
Comment (jamienoss) [inline ecl/hqlcpp/hqlttcpp.cpp]: Do we want a test for this or are there any that now hit this? I presume all the reg suits were run and passed ok?

**Comment by jamienoss** [comment]
@richardkchapman looks ok to me other than one question.
@ghalliday back to you briefly.

File: ecl/hqlcpp/hqlttcpp.cpp (hunk 1 above)
Comment (ghalliday) [inline ecl/hqlcpp/hqlttcpp.cpp]: It shouldn't be possibly to hit that line, so I'm not sure how I would write a test for it!
@richardkchapman

**Comment by jamienoss** [comment]
@richardkchapman this looks good to merge.

**Comment by richardkchapman** [comment]
Cherry-picked into 5.6.0

**Pull request merged**